budget-binder-multiplatform-app/src/commonMain/kotlin/de/hsfl/budgetBinder/data/repository/UserRepositoryImpl.kt

```diff
@@ -11,4 +11,12 @@ class UserRepositoryImpl(
     override suspend fun getMyUser(): APIResponse<User> {
         return client.getMyUser()
     }
+
+    override suspend fun changeMyUser(user: User.In): APIResponse<User> {
+        return client.changeMyUser(user)
+    }
+
+    override suspend fun removeMyUser(): APIResponse<User> {
```

Comment:
Was macht removeMyUser genau? ist es ein delete oder ein logout? Name wirkt auf mich gerade nicht eindeutig

Reply:
`remove...` ist vielleicht wirklich nicht der optimale Name. Die Funktion löscht den Benutzer. Hab jetzt aber für alle bereiche remove gewählt `removeCategory, removeEntry, unsw.`. Sollte aber wirklich in `delete...` umbenannt werden